dist/vue-popper.js

```diff
@@ -97,27 +97,27 @@
         currentPlacement: '',
         popperOptions: {
           placement: 'bottom',
-          computeStyle: {
-            gpuAcceleration: false
-          }
+          modifiers: [{
+            name: 'offset',
+            options: {
+              offset: [0, 8]
+            }
+          }, {
+            name: 'computeStyles',
+            options: {
+              gpuAcceleration: false // true by default
```

Comment:
@RobinCK do you recall why you originally set this to false? Find anything in git history or issues. Maybe we can remove this?

Reply:
I don't remember anymore, I think it’s possible to delete